### PR TITLE
[Doc] Running Profile document add HASH_JOIN_NODE, etc.

### LINKED ISSUE #4875: [Proposal] Running Profile document add HASH_JOIN_NODE, etc.
**Why Do So**
Currently, Running Profile document is missing some nodes, HASH_JOIN_NODE, CROSS_JOIN_NODE, UNION_NODE, etc.

** Solution**
Improve Running Profile document

### PR DESCRIPTION
**Pull request by xinyiZzz**
## Proposed changes

  - Running Profile document add `HASH_JOIN_NODE`, `CROSS_JOIN_NODE`, `UNION_NODE`, `ANALYTIC_EVAL_NODE`.
  - `UNION_NODE` increase`MaterializeExprsEvaluateTime` profile.

## Types of changes

- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have create an issue on (Fix #4875), and have described the bug/feature there in detail
- [x] If this change need a document change, I have updated the document

### CODE REVIEW

File: docs/zh-CN/administrator-guide/running-profile.md [hunk 1]
```diff
@@ -141,6 +145,33 @@ BE端收集的统计信息较多，下面列出了各个参数的对应含义：
...
+  - ProbeRows: 遍历左表进行Hash Probe的行数
+  - ProbeTime: 遍历左表进行Hash Probe的耗时，不包括对左表RowBatch调用GetNext的耗时
+  - PushDownComputeTime: 谓词下推条件计算耗时
+  - PushDownTime: 谓词下推的总耗时，Join时对满足要求的右表，转为左表的in查询
+
+#### `CROSS_JOIN_NODE`
+  - ExecOption: 对右表构造Build-side Table的方式（同步or异步）
+  - BuildRows: Build-side Table的行数
+  - BuildTime: 构造Build-side Table的耗时
+  - LeftChildRows: 遍历左表进行Probe的行数
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/running-profile.md]:
```suggestion
  - LeftChildRows: cross join 的左孩子的行数
```

File: docs/zh-CN/administrator-guide/running-profile.md [hunk 2]
```diff
@@ -141,6 +145,33 @@ BE端收集的统计信息较多，下面列出了各个参数的对应含义：
...
+  - ProbeTime: 遍历左孩子进行Hash Probe的耗时，不包括对左孩子RowBatch调用GetNext的耗时
+  - PushDownComputeTime: 谓词下推条件计算耗时
+  - PushDownTime: 谓词下推的总耗时，Join时对满足要求的右孩子，转为左孩子的in查询
+
+#### `CROSS_JOIN_NODE`
+  - ExecOption: 对右孩子构造RowBatchList的方式（同步or异步）
+  - BuildRows: RowBatchList的行数（即右孩子的行数）
+  - BuildTime: 构造RowBatchList的耗时
+  - LeftChildRows: 遍历左孩子进行Probe的行数
+  - LeftChildTime: 遍历左孩子进行Probe的耗时，不包括对左孩子RowBatch调用GetNext的耗时
```
Comment (EmmyMiao87) [inline docs/zh-CN/administrator-guide/running-profile.md:164]: 是左孩子通过probe 探针 遍历右表的耗时？

Comment (xinyiZzz) [reply docs/zh-CN/administrator-guide/running-profile.md:164]: Thanks, I have modified

**Review by EmmyMiao87 — APPROVED** [review]
LGTM

**Pull request merged**